tests/functional/test_install.py

```diff
@@ -49,12 +53,11 @@ def test_pip_second_command_line_interface_works(script):
     assert initools_folder in result.files_created, str(result)
 
 
-@pytest.mark.network
 def test_install_from_pypi(script):
     """
     Test installing a package from PyPI.
```

Comment:
hmm, just noticed the intent of this test is actually test PyPI.
maybe this one should stay until we have the local http index thing.